tutorials/haskell/hspec-golden-test-tutorial/code/package.yaml

```diff
@@ -0,0 +1,54 @@
+name:                hspec-golden-tests
+version:             0.1.0.0
+github:              "githubuser/hspec-golden-tests"
+license:             BSD3
+author:              "Author name here"
+maintainer:          "example@example.com"
+copyright:           "2019 Author name here"
+
+extra-source-files:
+- README.md
+- ChangeLog.md
```

Comment:
@Jagl257 I don't think we need a `ChangeLog.md`, do we?

Reply:
No we dont as I deleted the file.